Data/ByteString/FastBuilder/Internal.hs

```diff
@@ -153,7 +154,7 @@ data DynamicSink
       -- to a 'ThreadedSink'.
 
 -- | A mutable buffer. The 'Int' specifies where the data start.
-data Queue = Queue !(ForeignPtr Word8) !Int{-start-}
+data Queue = Queue !(ForeignPtr Word8) !Int{-start-} !Int{-total-}
```

Comment:
Perhaps it's time to change this datatype into a record? Also please update the comment.

Reply:
sure